crates/oxc_minifier/src/compressor/mod.rs

```diff
@@ -182,7 +182,9 @@ impl<'a> Compressor<'a> {
 
 impl<'a, 'b> VisitMut<'a, 'b> for Compressor<'a> {
     fn visit_statements(&mut self, stmts: &'b mut Vec<'a, Statement<'a>>) {
-        stmts.retain(|stmt| !self.drop_empty(stmt) && !self.drop_debugger(stmt));
+        if stmts.len() > 1 {
```

Comment:
I don't think this is correct, we can safely drop all statements and leave it as a empty vec.